src/commonMain/kotlin/character/classes/priest/pet/buffs/ManaLeech.kt

```diff
@@ -0,0 +1,33 @@
+package character.classes.priest.pet.buffs
+
+import character.*
+import character.classes.hunter.talents.FerociousInspiration
```

Comment:
I assume this import is not needed?  Effects like FI are applied to the entire raid, including their pets (even though an SPriest in a hunter group is highly unlikely).

Reply:
Yep, probably just copied over / missed it from another spell.

Reply:
Removed erroneous import